app/src/main/java/fr/free/nrw/commons/contributions/UploadCountClient.java

```diff
@@ -14,10 +14,10 @@
 import fr.free.nrw.commons.concurrency.ThreadPoolExecutorService;
 import timber.log.Timber;
 
-public class UploadCountClient {
+class UploadCountClient {
     private ThreadPoolExecutorService threadPoolExecutor;
 
-    public UploadCountClient() {
+    UploadCountClient() {
```

Comment:
Fair point, this is a fairly new class after all and I don't think any other package is likely to use it. I'm good with the change in that case. :)